symbols/overlay29.yml

```diff
@@ -5683,6 +5704,18 @@ overlay29:
         r3: Move pointer
         stack[0]: True to check all the possible move_ai_condition values, false to only check for move_ai_condition::AI_CONDITION_RANDOM (if the move has a different ai condition, the result will be false).
         return: True if the target is eligible, false otherwise
+    - name: TargetRegularAttack
+      address:
+        EU: 0x231AEC0
+        NA: 0x231A460
+        JP: 0x231B930
+      description: |-
+        Decides which direction the AI will use its regular attack in.
+        
+        r0: User entity pointer
+        r1: Output pointer to the direction the regular attack should be targeted at.
```

Comment:
This isn't reflected in the prototype's param type. Also if it's output, use the convention of `[output]`